src/libcore/fmt/mod.rs

```diff
@@ -803,7 +803,7 @@ pub trait UpperExp {
 /// assert_eq!(output, "Hello world!");
 /// ```
 ///
-/// [write_macro]: ../../std/macro.write!.html
+/// [write_macro]: ../../std/macro.write.html
```

Comment:
Should this be changed to "[`write!`]" instead of [write_macro] while we're here?